server/db/helper.js

```diff
@@ -114,6 +114,25 @@ const getUserBadges = (userId) => UserBadge.findAll({
 });
 
 
+const getCompletedCourse = (userId) => UserBadge.findAll({
+  where: {
+    idUser: userId
+  },
+
+  attributes: ['idBadge']
+})
+.then((badgesId) => {
+  let courseId = badgesId.map((badgeId) => badgeId.dataValues.idBadge)
```

Comment:
Can you change this variable name as well and make it plural since it is not an array of course ids but an array of badge ids

Reply:
What is the intention of this helper? To get a single course completed or all courses the user has completed?
It looks like we're getting all badges of a user (so all courses) and but then the variables get singular. I'd go ahead an add pseudo code to this as well for documentation